<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScaleDialog</class>
 <widget class="QDialog" name="ScaleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map Scale</string>
  </property>
  <property name="styleSheet">
   <string> QRadioButton::hover {
	background-color: rgb(188, 201, 255);
 }
QRadioButton::checked {
	background-color: rgb(159, 166, 235);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Please select the map scale to use:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="scale0Btn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/scale_0.png</normaloff>:/images/scale_0.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="scale1Btn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/scale_1.png</normaloff>:/images/scale_1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="scale2Btn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/scale_2.png</normaloff>:/images/scale_2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QRadioButton" name="scale3Btn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/scale_3.png</normaloff>:/images/scale_3.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="scale4Btn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/scale_4.png</normaloff>:/images/scale_4.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="scale5Btn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/scale_5.png</normaloff>:/images/scale_5.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QRadioButton" name="scale6Btn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/scale_6.png</normaloff>:/images/scale_6.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QRadioButton" name="scale7Btn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/images/scale_7.png</normaloff>:/images/scale_7.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ScaleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ScaleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
